riak/transports/feature_detect.py

```diff
@@ -192,6 +194,22 @@ def datatypes(self):
         """
         return self.server_version >= versions[2.0]
 
+    def preflists(self):
+        """
+        Whether bucket/key preflists are supported.
+
+        :rtype: bool
+        """
+        return self.server_version >= versions[2.1]
```

Comment:
Should this also check for security and, if enabled, require a version greater than 2.1.1 ?